src/wp-includes/html-api/class-wp-html.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * HTML API: WP_HTML class
+ *
+ * Provides a public interface for HTML-related functionality in WordPress.
+ *
+ * @package WordPress
+ * @subpackage HTML-API
+ * @since 6.5.0
+ */
```

Comment:
I've used `WP_HTML_Processor::is_void_tag` a few times and it feels like it should live on a utility class like this.

Reply:
it started in a utility class like this! but then we didn't include the utility class for pragmatic reasons.